src/streams/index.ts

```diff
@@ -4,4 +4,4 @@ export * from "./readEventsFromStream";
 export * from "./subscribeToAll";
 export * from "./subscribeToStream";
 export * from "./tombstoneStream";
-export * from "./writeEventsToStream";
+export * from "./appendEventsToStream";
```

Comment:
Index exports should be alphabetical, but its not enforced and probably not followed everywhere

Reply:
Updated in: https://github.com/EventStore/EventStore-Client-NodeJS/pull/88/commits/b3ea9b4236373d9c58dcd2839977ab9041482eb9.